### PR TITLE
Find the paraview python libraries for the linux binaries

### PR DESCRIPTION
**Pull request by mathturtle**
We have to add the path to the paraview python packages to the python path.

### CODE REVIEW

File: tomviz/tomvizPythonConfig.h.in [hunk 1]
```diff
@@ -91,6 +93,10 @@ void PythonAppInitPrependPathLinux(const std::string& exe_dir)
     PythonAppInitPrependPythonPath(exe_dir + "/../" TOMVIZ_PYTHON_INSTALL_DIR);
 
     // we don't add anything for ParaView since, ParaView takes care of that.
```
Comment (cryos) [inline tomviz/tomvizPythonConfig.h.in]: This begs the question of why ParaView is not taking care of this on Linux. If it is not (but is on other platforms) we should file a bug, and change this comment accordingly.

**Comment by mathturtle** [comment]
Updated the comment to explain why it is necessary after digging into ParaView to find out what wasn't working.

**Comment by cryos** [comment]
Typos that I would like you to fix, then feel free to merge whenever you are ready.

**Comment by mathturtle** [comment]
I'm going to merge this and the superbuild branch now and hopefully see if anything breaks by the end of the day.

**Pull request merged**